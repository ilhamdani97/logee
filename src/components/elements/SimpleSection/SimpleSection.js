import React, { Fragment } from 'react';
import PropTypes from 'prop-types';
import { Col, Row, Typography } from 'logeeport-ui';
import styles from './styles.scoped.css';

const SimpleSection = ({ title,className,children }) => {
  return (
    <Row className={[styles.root,className].join(' ')}>
      <Col className={styles.noPadSide} xs="12">
        <Typography className={styles.title} variant="h5">{title}</Typography>
      </Col>
      <Fragment>
        {children}
      </Fragment>
    </Row>);
};

SimpleSection.defaultProps = {
  children:null,
  className: '',
  title : '',


};

SimpleSection.propTypes = {
  children: PropTypes.node,
  className: PropTypes.string,
  title : PropTypes.string,
};


export default SimpleSection;
