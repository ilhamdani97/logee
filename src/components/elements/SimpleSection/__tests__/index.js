import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import SimpleSection from '../SimpleSection';

describe('src/components/elements/SimpleSection', () => {
  test('render', () => {
    const shallow = new ShallowRenderer();
    const tree = shallow.render(<SimpleSection title="asdasd"  />);
    expect(tree).toMatchSnapshot();
  });
});
