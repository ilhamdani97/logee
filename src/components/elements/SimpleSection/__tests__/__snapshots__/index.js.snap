// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/components/elements/SimpleSection render 1`] = `
<Row
  alignContent=""
  alignItem=""
  className=" "
  direction=""
  id=""
  justify=""
  noGutter={false}
>
  <Col
    className=""
    id=""
    lg=""
    md=""
    offsetLg=""
    offsetMd=""
    offsetSm=""
    offsetXL=""
    offsetXs=""
    orderLg=""
    orderMd=""
    orderSm=""
    orderXl=""
    orderXs=""
    sm=""
    xl=""
    xs="12"
  >
    <Typography
      spacing=""
      variant="h5"
    >
      asdasd
    </Typography>
  </Col>
  <React.Fragment />
</Row>
`;
